tools/cluster-init.in

```diff
@@ -84,6 +84,16 @@ function helptext() {
     exit $1
 }
 
+function sed_in_place() {
+    cp $1 $1.$$
+    sed -e "$2" $1 > $1.$$
+    mv $1.$$ $1
+}
+
+function sed_in_place_remotely() {
+    ssh -l root $1 -- cp $2 $2.sed \&\& sed -e "$3" $2 \> $2.sed \&\& mv $2.sed $2
+}
+
```

Comment:
This will cascade up to getting rid of the --setup argument to cts/cts, which obviously isn't a big deal, but makes me start wondering how useful the clean and install arguments are as well.

Reply:
clean/install should still be usable